.github/workflows/unit_test_bcc.yml

```diff
@@ -8,5 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+    - name: install linux headers
```

Comment:
@marceloamaral  by this test case, which means the unit test will run without bcc build tag for golang.